filebeat/module/pensando/_meta/kibana/7/dashboard/pensando-dfw-overview.json2

```diff
@@ -0,0 +1,1390 @@
+{
```

Comment:
I think this file is duplicated and probably needs to be removed.

Reply:
removed in 6e07cd936897d7aa1351515e1b573668e826b35d